tests/test_imports.py

```diff
@@ -0,0 +1,43 @@
+import platform
+import sys
+
+import pytest
+
+
+def test___all__(pytester: pytest.Pytester) -> None:
+    """See https://github.com/aio-libs/aiohttp/issues/6197"""
+    pytester.makepyfile(
+        test_a="""
+            from aiohttp import *
+            assert 'GunicornWebWorker' in globals()
+        """
+    )
+    result = pytester.runpytest("-vv")
+    result.assert_outcomes(passed=0, errors=0)
+
+
+def test_web___all__(pytester: pytest.Pytester) -> None:
+    pytester.makepyfile(
+        test_b="""
+            from aiohttp.web import *
+        """
+    )
+    result = pytester.runpytest("-vv")
+    result.assert_outcomes(passed=0, errors=0)
+
+
+@pytest.mark.skipif(
+    not sys.platform.startswith("linux") or platform.python_implementation() == "PyPy",
+    reason="Timing is less reliable on other platforms",
```

Comment:
Plz. Specify which platforms. Take into account that this string shows up in the test report.
Imaging seeing "SKIPPED: Timing is less reliable on other platforms" in the logs under macos — would it be helpful? Would you immediately guess that you're on an "other" platform?